kind/config/overrides.yaml

```diff
@@ -0,0 +1,113 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: installation-config-overrides
+  namespace: kyma-installer
+  labels:
+    installer: overrides
+    kyma-project.io/installation: ""
+data:
+  global.isLocalEnv: "true"
```

Comment:
Maybe some kind of comment is needed here? You've spent a lot of our meeting's time here